api/proto/external.proto

```diff
@@ -603,10 +603,31 @@ message UnsignedTx {
     repeated InputRing rings = 2;
 
     /// The amount and blinding factors of each of the outputs we are creating.
-    repeated OutputSecret output_secrets = 3;
+    repeated OutputSecret output_secrets = 3 [deprecated = true];
```

Comment:
@briancorbin they can get all the same data they had in this field from the `tx_out_summary_unblinding_data.unmasked_amount`

we could rename this to just `tx_out_unblinding_data` I guess

Reply:
Sounds good to me

I think copper doesn't actually use this, I think Brian's full-service offline signing flow is the only thing that uses this.